documentation/src/main/asciidoc/manual-index.asciidoc

```diff
@@ -398,6 +401,114 @@ long executionId = BatchRuntime
 ----
 ====
 
+==== Job Parameters
+
+.Job Parameters in JSR 352 Integration
+|===
+|Parameter Name |Builder Method |Requirement |Default value |Description
+
+|`rootEntities`
+|`forEntity(Class<?>)`, `forEntities(Class<?>, Class<?>...)`
+|Required
+|-
+|The root entity types to index in this job execution.
+
+|`purgeAllOnStart`
+|`purgeAllOnStart(boolean)`
+|Optional
+|False
+|Specify whether the existing lucene index should be purged at the beginning of the job. This
+operation takes place before the step of lucene document production.
+
+|`optimizeAfterPurge`
+|`optimizeAfterPurge(boolean)`
+|Optional
+|False
+|Specify whether the mass indexer should be optimized at the beginning of the job. This operation
+takes place after the purge operation and before the step of lucene document production.
+
+|`optimizeOnFinish`
+|`optimizeOnFinish(boolean)`
+|Optional
+|False
+|Specify whether the mass indexer should be optimized at the end of the job. This operation takes
+place after the step of lucene document production.
+
+|`cacheable`
+|`cacheable(boolean)`
+|Optional
+|False
+|Specify whether the Hibernate queries are cacheable. This setting will be applied to the internal
+entity reader class. Set it to true when reading a complex graph with relations.
+
+|`fetchSize`
+|`fetchSize(int)`
+|Optional
+|200,000
+|The size for the result fetching.
+
+|`customQueryHQL`
+|`restrictedBy(String)`
+|Optional
+|-
+|Use HQL / JPQL to select the entities to index. Mixing this approach with the criteria restriction
+is not allowed. Please notice that there's no query validation for your input.
+
+|`customQueryCriteria`
```

Comment:
I agree with you that the parameter `customQueryCriteria` is not well placed here. However, I'm not sure how to modify it, because the cheat sheet should be a quick reference for launching the job via CLI and via query builder, am I right? So I don't know how to stating it off properly.

Could you help me fix it up?